canonical-websites/snapcraft.io
# Snap details map

Adds a map of snap usage to snap details page.

## QA
- ./run
- Go to snap page
- Map should appear

Or try demo:
http://snapcraft.io-pr-71.run.demo.haus/lxd/
http://snapcraft.io-pr-71.run.demo.haus/gnome-calculator/
http://snapcraft.io-pr-71.run.demo.haus/canonical-livepatch/
http://snapcraft.io-pr-71.run.demo.haus/core/


<img width="1012" alt="screen shot 2017-10-12 at 15 13 40" src="https://user-images.githubusercontent.com/83575/31497692-22977726-af60-11e7-9daa-34cf4ffc8e66.png">
